src/components/AdminPage.js

```diff
@@ -1,3 +1,6 @@
-export default function AdminPage() {
-  return 'Admin Page...';
-}
+import React from 'react';
+import Layout from './AdminPage/AdminLayout';
+
+export default () => (
+  <Layout />
```

Comment:
This seems redundant to me. Why not change just import the `Layout` into the Router instead of importing `AdminPage`?